### PR TITLE
add pyincore-data instruction

### PR DESCRIPTION
**Pull request by longshuicy**
Add pyincore-data installation instruction per user request

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to the INCORE documents generated by Sphinx package will be
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Added
+- Instruction to install pyincore-data [#412](https://github.com/IN-CORE/pyincore/issues/412)
```
Comment (navarroc) [inline CHANGELOG.md:10]: This should link to docs

File: manual_jb/content/getting_started.md [hunk 2]
```diff
@@ -85,13 +85,26 @@ If you don't have Miniconda installed, do the following steps.
...
+    ```
+   
+   If the installed pyincore, pyincore-viz, pyincore-data version is not the latest or lower than the desired one, 
+   specify the version number in installation command. e.g.
     ```
     conda install -c in-core pyincore=1.14.0 (or your version of choice)
     ```
-   Version information for pyincore and pyincore-viz can be found in
+   Version information for pyincore, pyincore-viz, and pyincore-data can be found in
    - https://anaconda.org/IN-CORE/pyincore
```
Comment (navarroc) [inline manual_jb/content/getting_started.md:105]: Can you make each of these hyperlinks? I think this was an oversight when we originally did the page.

**Comment by navarroc** [comment]
Overall this looks good. Just one suggestion about updating the conda package links to hyperlinks (not required, but nice to have) and then the CHANGELOG issue should link to incore-docs.

**Review by navarroc — APPROVED** [review]

**Comment by ywkim312** [comment]
I will fix the merge conflict and will merge

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**